src/mcu_node/main.cpp

```diff
@@ -148,6 +148,70 @@ void sendSDO(int nodeID, uint16_t index, uint8_t subindex, uint32_t data)
     sendCANOpenPacket(R_SDO, nodeID, buffer);
 }
 
+// Function, sendSDO, sends data of one byte in size.
+void sendSDO(int nodeID, uint16_t index, uint8_t subindex, uint8_t data, bool singleByte) {
```

Comment:
What is `bool singleByte` used for?

Reply:
valid; it's also set as true everywhere you guys call it if that's of import.